src/test/scala/org/broadinstitute/dsde/vault/services/uBAM/UBamDescribeServiceSpec.scala

```diff
@@ -78,11 +80,21 @@ class UBamDescribeServiceSpec extends VaultFreeSpec with UBamDescribeService wit
             }
           }
         }
-
+        "when calling list  UBam Describe path " - {
```

Comment:
1. This test is failing on 91, as the route isn't being handled when `version == None`. Up to you if you want test for `handled == false`, adjust the route to actually return a forced 404, or otherwise to fix the test.
2. The spec behavior on 84 should probably be a little clearer than "should return a list". Maybe "should return a list or not be handled"?
3. Extra space between "list" and "UBam" in the outer spec behavior on 83.